.vscode/scripts/import-example-app.sh

```diff
@@ -27,6 +27,9 @@ else
   rm -rf app/
   cp -a $VELOCITAS_EXAMPLES_PATH/$CHOSEN_EXAMPLE/. app/
 
+  pip-compile ./app/requirements.txt
```

Comment:
please check how to use pip-compile: we want to rebuild the requirements.txt in the root of the repo